src/main/java/net/dv8tion/jda/core/entities/Member.java

```diff
@@ -62,15 +98,66 @@
      */
     String getNickname();
 
+    /**
+     * Retrieves the Name displayed in the official Discord Client.
+     *
+     * @return
+     *      The Nickname of this Member or the Username if no Nickname is present.
+     */
     String getEffectiveName();
 
+    /**
+     * The roles applied to this Member.
+     *
+     * @return
+     *      An immutable List of {@link net.dv8tion.jda.core.entities.Role Roles} for this Member.
+     */
     List<Role> getRoles();
 
+    /**
+     * An {@link java.awt.Color Color} this Member is displayed with.<br>
+     * Retrieved by the color determining Role this Member has.
+     *
+     * @return
+     *      The display Color for this Member.
+     */
     Color getColor();
 
+    /**
+     * The Guild-Wide Permissions this Member holds.
+     *
+     * @return
+     *      An immutable List of Permissions granted to this Member.
+     */
     List<Permission> getPermissions();
+
+    /**
+     * The Permissions this Member holds in the specified {@link net.dv8tion.jda.core.entities.Channel Channel}.
+     *
```

Comment:
add: Permissions returned by this may be different from {#getPermissions()} due to the Channel's {PermissionOverrides}